### PR TITLE
New Unit Tests

### PR DESCRIPTION
**Pull request by marshl**
Added units tests for the new functions in the Trial models (and fixed a few issues found when writing the unit tests)

### CODE REVIEW

File: tests/unit/models/TrialTest.php [hunk 1]
```diff
@@ -20,6 +20,11 @@ public function setUp()
         parent::setUp();
     }
 
+    public function tearDown()
```
Comment (apigram) [inline tests/unit/models/TrialTest.php]: You probably don't need to override setUp and tearDown if you aren't adding any extra processing to it.

File: views/trial/permissions.php [hunk 2]
```diff
@@ -192,8 +192,12 @@ function removeSelectedUser() {
               new OpenEyes.UI.Dialog.Alert({
                 content: "That patient has already been shared to this trial. To change their permissions, please remove them first and try again."
               }).open();
-            } else {
+            } else if(html === '<?php echo Trial::RETURN_CODE_USER_PERMISSION_OK; ?>') {
```
Comment (apigram) [inline views/trial/permissions.php]: Do spaces work here? I thought in PHP it was elseif and not else if?

File: tests/unit/models/TrialTest.php (hunk 1 above)
Comment (marshl) [inline tests/unit/models/TrialTest.php]: fair point, I think I just copied that from the sample. It shall be removed

File: views/trial/permissions.php (hunk 2 above)
Comment (marshl) [inline views/trial/permissions.php]: The "else if" here is in JavaScript not PHP. That said, I will fix the whitespace issue.

Comment (apigram) [inline views/trial/permissions.php]: Very well then. Feel free to discard my comment.

**Pull request merged**